### PR TITLE
Add support of Healthcheck's

### PR DESCRIPTION
**Pull request by cono**

### CODE REVIEW

**Comment by fstehle** [comment]
Hey @cono, thanks a lot for your contribution.
Can you please rebase your branch onto master to resolve the conflicts?

**Comment by cono** [comment]
done

**Comment by cono** [comment]
looks like flake8 doesn't like my code :)

File: library/fastly_service.py [hunk 1]
```diff
@@ -375,6 +381,40 @@ def sort_key(f):
         return f.name
 
 
+class FastlyHealthcheck(FastlyObject):
+    schema = {
+        'name': dict(required=True, type='str', default=None),
+        'check_interval': dict(required=False, type='int', default=None),
+        'comment': dict(required=False, type='str', default=''),
+        'expected_response': dict(required=False, type='int', default='200'),
```
Comment (fstehle) [inline library/fastly_service.py]: 200 should be an integer value (no quotes)

**Pull request merged**